readhomer_atlas/library/schema.py

```diff
@@ -257,8 +262,20 @@ def get_queryset(cls, queryset, info):
 
     def resolve_metadata(obj, *args, **kwargs):
         metadata = obj.metadata
+        work = obj.get_parent()
+        text_group = work.get_parent()
+        # @@@ backport lang map
+        lang_map = {
+            "eng": "English",
+            "grc": "Greek",
+        }
         metadata.update(
-            {"work_urn": URN(metadata["first_passage_urn"]).up_to(URN.WORK)}
+            {
+                "work_label": work.label,
+                "text_group_label": text_group.label,
+                "lang": metadata["lang"],
+                "human_lang": lang_map[metadata["lang"]],
```

Comment:
wondering if we should do a `.get()` with a fallback here (e.g. "Unknown")

Reply:
it might not be a big deal because it would just presumably fail to ingest (so failing on bad lang tag might not be a bad thing)

Reply:
As discussed in Slack, we'll hardcode this for now and revisit as part of ![](https://github.trello.services/images/mini-trello-icon.png)  [Standardize language codes within ATLAS](https://trello.com/c/VZYbpvle)